src/routes/documents/[id]-[slug]/sidebar/Actions.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import type { Document } from "$lib/api/types";
+
+  import {
+    Apps16,
+    Comment16,
+    Download16,
+    EyeClosed16,
+    History16,
+    Lock16,
+    Pencil16,
+    Share16,
+  } from "svelte-octicons";
+
+  import Action from "$lib/components/common/Action.svelte";
+  import Flex from "$lib/components/common/Flex.svelte";
+  import SignedIn from "$lib/components/common/SignedIn.svelte";
+  import SidebarItem from "$lib/components/sidebar/SidebarItem.svelte";
+
+  import { canonicalUrl, pdfUrl } from "$lib/api/documents";
+
+  export let document: Document;
+
+  function relative(document: Document, path: string) {
+    return new URL(path, canonicalUrl(document)).toString();
+  }
+
+  // urls
+  $: edit = relative(document, "edit/");
+  $: revisions = relative(document, "revisions/");
+  $: pdf = pdfUrl(document).toString();
+  $: annotate = relative(document, "annotate/");
+  $: redact = relative(document, "redact/");
+  $: modify = relative(document, "modify/");
+</script>
+
+<Flex direction="column">
+  <SidebarItem>
+    <Lock16 />
+    <span class="access">{document.access} access</span>
+
+    <SignedIn>
+      <Action icon={Pencil16}><a href={edit}>Edit</a></Action>
+    </SignedIn>
+  </SidebarItem>
+
+  <SidebarItem>
+    <History16 />
+    <a href={revisions}>Revision History</a>
+  </SidebarItem>
+
+  <SidebarItem>
+    <Download16 />
+    <a target="_blank" rel="noopener noreferrer" href={pdf}>Download PDF</a>
+  </SidebarItem>
```

Comment:
This is good for now, but I think we'll need to rework the SidebarItem slightly to handle links. I think the entire "row" should behave like a link when that's the action it represents.